lib/trento/discovery/policies/cluster_policy.ex

```diff
@@ -231,13 +231,26 @@ defmodule Trento.Discovery.Policies.ClusterPolicy do
 
       site = Map.get(attributes, "hana_#{String.downcase(sid)}_site", "")
 
+      hana_roles =
+        attributes
```

Comment:
you could even pipe this to `parse_nodes_actual_roles`

Reply:
nice catch, done